### PR TITLE
add extended statistics doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
add extended statistics doc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/9891
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: extended-statistics.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+本文介绍如何使用扩展统计信息来指导 SQL 优化。TiDB 可以收集以下两种统计信息：
+
+- 统计信息：如直方图和 Count-Min Sketch 等。更多详细信息，请参阅[统计信息](/statistics.md)。
+- 扩展统计信息：按表和列过滤的统计信息。
+
+> **建议：**
+>
+> 在阅读本文档之前，建议先阅读[使用统计信息](/statistics.md)。
+
+手动或自动执行 `ANALYZE` 语句时，TiDB 默认只收集常规统计信息，不收集扩展统计信息。这是因为扩展统计信息仅用于特定场景下的优化器估算，而且收集扩展统计信息会造成额外的开销。
```
Comment (songrijie) [inline extended-statistics.md:17]: 前面标题里把”常规统计信息“叫”基础统计信息“，这里要统一一下叫法。

**Comment by songrijie** [comment]
rest LGTM

File: extended-statistics.md (hunk 1 above)
Comment (TomShawn) [reply extended-statistics.md:17]:
```suggestion
手动或自动执行 `ANALYZE` 语句时，TiDB 默认只收集基础统计信息，不收集扩展统计信息。这是因为扩展统计信息仅用于特定场景下的优化器估算，而且收集扩展统计信息会造成额外的开销。
```

File: extended-statistics.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
+---
+title: 使用扩展统计信息
+summary: 了解如何使用扩展统计信息来指导 SQL 优化。
+---
+
+# 使用扩展统计信息
+
+本文介绍如何使用扩展统计信息来指导 SQL 优化。TiDB 可以收集以下两种统计信息：
+
+- 统计信息：如直方图和 Count-Min Sketch 等。更多详细信息，请参阅[统计信息](/statistics.md)。
```
Comment (TomShawn) [inline extended-statistics.md:10]:
```suggestion
- 基础统计信息：如直方图和 Count-Min Sketch 等。更多详细信息，请参阅[统计信息](/statistics.md)。
```

File: extended-statistics.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+- 扩展统计信息：按表和列过滤的统计信息。
+
+> **建议：**
+>
+> 在阅读本文档之前，建议先阅读[使用统计信息](/statistics.md)。
```
Comment (TomShawn) [inline extended-statistics.md:15]:
```suggestion
> 在阅读本文档之前，建议先阅读[基础统计信息](/statistics.md)。
```

File: extended-statistics.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+手动或自动执行 `ANALYZE` 语句时，TiDB 默认只收集基础统计信息，不收集扩展统计信息。这是因为扩展统计信息仅用于特定场景下的优化器估算，而且收集扩展统计信息会造成额外的开销。
+
+默认情况下，TiDB 不收集扩展统计信息。要使用扩展统计信息，你需要先启用扩展统计信息，然后单独注册每个扩展统计信息对象。
+
+注册完成后，再执行 `ANALYZE` 语句时，TiDB 会同时收集常规统计信息和注册的扩展统计信息。
```
Comment (TomShawn) [inline extended-statistics.md:21]:
```suggestion
注册完成后，再执行 `ANALYZE` 语句时，TiDB 会同时收集基础统计信息和注册的扩展统计信息。
```

**Comment by qiancai** [comment]
Removed the **needs-cherry-pick-release-6.3** label because the v6.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.3 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.6 label because the v6.6 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.6 and will no longer receive new updates.

**Comment by lilin90** [comment]
Note: Since many conflicts exist in this PR after a long period of time, and some descriptions are outdated compared with the English version, I'll close this PR and create another one later.

**Pull request closed without merging**